README.md

```diff
@@ -1143,6 +1147,51 @@ Gatsby does this in an effort to reduce the likelihood installing one plugin can
 
 You can read more about this if you’re interested in [this issue](https://github.com/imgix/gatsby/issues/129).
 
+### Multiple imgix Sources
+
+You might find yourself with multiple imgix sources and wondering how you could use them at the same time with this plugin. There are a few possibilities for this, which will be outlined below.
+
+Any number of sources can be used simultaneously with the URL transform API. The caveats with this approach is that you don't get the benefits of the GraphQL APIs (blur-up, etc), and that the sources must not be Web Proxy sources (for these you must use one of the GraphQL APIs).
```

Comment:
The only nit is that maybe it's better to add the `Only caveat` section after the example. This way I can _see_ what you mean a little sooner. But that's a personal preference, so feel free to ignore it.

Reply:
I can get behind this